models/index.ts

```diff
@@ -1,43 +1,141 @@
-'use strict';
-
-const fs = require('fs');
-const path = require('path');
-const Sequelize = require('sequelize');
-const process = require('process');
-const basename = path.basename(__filename);
-const env = process.env.NODE_ENV || 'development';
-const config = require(__dirname + '/../config/config.js')[env];
-const db: any = {};
-
-let sequelize: any;
-if (config.use_env_variable) {
-  sequelize = new Sequelize(process.env[config.use_env_variable], config);
-} else {
-  sequelize = new Sequelize(config.database, config.username, config.password, config);
-}
+import sequelize from './sequelize';
+
+import Brands from './brands';
+import Categories from './categories';
+import Addresses from './addressess';
+import Users from './users';
+
+import Cart from './cart';
+import CartProducts from './cartproducts';
+import CartProductsVariationsOptions from './cartproductsvariationsoptions';
+
+import Orders from './orders';
+import Products from './products';
+
+import Variation from './variations';
+import VariationOption from './variationsoptions';
+import Wishing from './wishing';
+
+
+// brands
+Brands.hasMany(Products, {
```

Comment:
Change the relations setup here by adding the tables that doesn't have any foreign-key first. Why? because the other-table will be required to create the current.